perf_bench_test.go

```diff
@@ -695,36 +697,14 @@ func bindProcessToCPU(pid string, processors ...int) error {
 		procStr += sep + strconv.Itoa(proc)
```

Comment:
in the code above, you used the term `cpus`, here it is `processors`. Can you please unify the terminology?

Also, please add a comment, describing what this function does.

Reply:
Done